src/cript/api/schema_validation.py

```diff
@@ -21,7 +23,7 @@ def _get_db_schema() -> json:
     json
         The database schema in JSON format.
     """
-    host: str = "https://development.api.criptapp.org"
+    host: str = _global_cached_api.host
```

Comment:
_global_cached_api can't be null?

Reply:
no it can.
@nh916 please use `_get_global_cache_api()` instead.
That catches that problem.

good catch @bearmit

Reply:
Thank you so much @InnocentBug for updating this and making the push for me! 🙏